Sources/ReactorKit/Reactor.swift

```diff
@@ -71,60 +71,52 @@ public protocol Reactor: AnyObject {
 private typealias AnyReactor = AnyObject
 
 private enum MapTables {
-  static let action = WeakMapTable<AnyReactor, AnyObject>()
+  static let streams = WeakMapTable<AnyReactor, AnyObject>()
   static let currentState = WeakMapTable<AnyReactor, Any>()
-  static let state = WeakMapTable<AnyReactor, AnyObject>()
   static let disposeBag = WeakMapTable<AnyReactor, DisposeBag>()
   static let isStubEnabled = WeakMapTable<AnyReactor, Bool>()
   static let stub = WeakMapTable<AnyReactor, AnyObject>()
 }
 
 
+// MARK: - ReactorStreams
+
+private struct ReactorStreams<Action, State> {
+  let action: ActionSubject<Action>
+  let state: Observable<State>
+}
+
 // MARK: - Default Implementations
 
 extension Reactor {
-  private var _action: ActionSubject<Action> {
-    if isStubEnabled {
-      return stub.action
+  private var streams: ReactorStreams<Action, State> {
+    if (isStubEnabled) {
```

Comment:
```suggestion
    if isStubEnabled {
```
swift prefer convention that does not use brackets in the conditional state.

Reply:
Modified! 👍